examples/franka_blockstacking_example.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+
+import numpy as np
+from display_utils import display_rgb
+
+import habitat
+from habitat.core.logging import logger
+
+SENSOR_KEY = "rgb"
+CONFIG_FILE = "configs/tasks/franka_point.yaml"
+BLOCK_LOC_RANGE = 2.0
+BLOCK_LOC_CENTER = [-5.0, 0.1, 8.5]
+
+
+def example(render):
+    # Note: Use with for the example testing, doesn't need to be like this on the README
+
+    with habitat.Env(config=habitat.get_config(CONFIG_FILE)) as env:
+        logger.info("Environment creation successful")
+
+        obj_templates_mgr = env.sim.get_object_template_manager()
+        rigid_obj_mgr = env.sim.get_rigid_object_manager()
+
+        cube_handle = obj_templates_mgr.get_template_handles("cubeSolid")[0]
+        cube_template_cpy = obj_templates_mgr.get_template_by_handle(
+            cube_handle
+        )
+        cube_template_cpy.scale = np.array([0.2, 0.2, 0.2])
+        obj_templates_mgr.register_template(
+            cube_template_cpy, "my_scaled_cube"
+        )
+
+        for _ in range(6):
+            obj = rigid_obj_mgr.add_object_by_template_handle("my_scaled_cube")
+            offset = (np.random.random(3) * 2 - 1) * BLOCK_LOC_RANGE
```

Comment:
To be more illustrative let's create consts for "2" and "-1" numbers to describe where it comes from.

Reply:
This line specifies that the blocks should be placed randomly and uniformly around BLOCK_LOC_CENTER, so I think making these additional constants may make actually obfuscate the intention. Do you think the following is clearer or do you still think that I should make the 2 and -1 new constants?

`offset = np.random.random(3) * BLOCK_LOC_RANGE * 2 - BLOCK_LOCK_RANGE`